conf/example1.conf

```diff
@@ -0,0 +1,48 @@
+http {
+    server {
+        listen 80
+        server_name first
+        index multi_form_upload.html
+        root /var/www/
+        location /about/ {
+            root /public/www/
+            index about.html
+            limit_except GET POST
+            client_max_body_size 75000
+        }
+    }
+    server {
+        listen 81
+        server_name second
+        error_pages 404 /www/error_pages/404.html
+        error_pages 401 /www/error_pages/401.html
+        error_pages 500 /www/error_pages/500.html
+        root /var/www/
+        location /about2/ {
+            root /public/www/
+            index about.html
+            limit_except GET POST
+            client_max_body_size 75000
+        }
+        location /about1/ {
+            root /public/www/
+            index about.html
+            limit_except GET POST
+            client_max_body_size 75000
+        }
+        location /about3/ {
+            root /public/www/
+            index about.html
+            limit_except GET POST
+            client_max_body_size 75000
+        }
+    }
+    server {
+        listen 81
```

Comment:
What happens if, like in this case, the PORT is already used? All the below config details are ignored?

Reply:
Error will be thrown and the server will say it can't start because the config is incorrect.